salt/pillar/data-pipeline.sls

```diff
@@ -22,6 +22,7 @@ data_pipeline:
 
     bigquery_views:
         revision: latest
+        materialize_arguments:
```

Comment:
I think you want an explicit empty string `""` as the default, otherwise this may gets printed as `None` when you use it

Reply:
actually, shouldn't the current default be `--disable-view-name-mapping`?

Reply:
Whatever you choose, we need to put it in https://github.com/elifesciences/builder-configuration/pull/95/files#diff-8e1abcc9fe49effc9fc073d53fdaa7edR25

Reply:
> actually, shouldn't the current default be `--disable-view-name-mapping`?

current default is to use the mapping,  `--disable-view-name-mapping` needs to be  explicitly specified in order to use it

Reply:
> > actually, shouldn't the current default be `--disable-view-name-mapping`?
> 
> current default is to use the mapping, `--disable-view-name-mapping` needs to be explicitly specified in order to use it

What I meant is that currently, we only want to use the mapping in production (otherwise we'd end up with the issue we currently have). That is why in terms of deployment configuration, we should pass `--disable-view-name-mapping` by default, except for the `prod` configuration.